src/lib.rs

```diff
@@ -7,17 +7,23 @@ mod core;
 mod crypto;
 mod error;
 
-pub use crate::core::{
-    cigar::Cigar,
-    counter::{tables as counter, Counter}, // This seems like it shoudl be an abstract class
-    diger::Diger,
-    indexer::tables as indexer,
-    matter::tables as matter,
-    prefixer::Prefixer,
-    saider::Saider,
-    seqner::Seqner,
-    siger::Siger,
-    signer::Signer,
-    util,
-    verfer::Verfer,
+pub use crate::{
+    core::{
+        cigar::Cigar,
+        counter::{tables as counter, Counter}, // This seems like it shoudl be an abstract class
+        diger::Diger,
+        indexer::{tables as indexer, Indexer},
+        matter::{tables as matter, Matter},
+        number::{tables as number, Number},
+        prefixer::Prefixer,
+        sadder::Sadder,
+        saider::Saider,
+        seqner::Seqner,
+        siger::Siger,
+        signer::Signer,
+        util,
```

Comment:
Also not sure we need this.

Reply:
Could be useful in parside I suppose.

Reply:
After talking to @m00sey decided to remove this, to minimize the surface we expose